### PR TITLE
Coolness added

### PR DESCRIPTION
**Pull request by arthar360**
I have added a cool comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# bsjava
\ No newline at end of file
+# bsjava
+
+This is a cool project
```
Comment (Steiner1729) [inline README.md:3]: Can you change this to cool college project?

Comment (Steiner1729) [reply README.md:3]: Add as a single comment

**Comment by arthar360** [comment]
done

**Review by Steiner1729 — APPROVED** [review]
The changes you made are approved

**Pull request merged**